nbconvert/filters/markdown.py

```diff
@@ -102,4 +102,4 @@ def rawlatex2math(text):
         json_data = json.loads(text, object_hook=rawlatex2math_hook)
         return json.dumps(json_data)
 
-    return pandoc(source, 'markdown', 'rst', extra_args=extra_args)
+    return pandoc(source, 'markdown', 'rst', extra_args=extra_args, filter_func=rawlatex2math)
```

Comment:
This doesn't have def to refer back to as far as I can tell. no wonder it doesn't work.